### PR TITLE
Histograms in Trees

### PR DESCRIPTION
**Pull request by reikdas**

### CODE REVIEW

**Comment by reikdas** [comment]
I am able to see the code changes in the github diff. Check - https://github.com/scikit-hep/uproot/pull/78/files#diff-26c1418a30d1cd262d87e9dfe59ca762R128

File: tests/test_tree.py [hunk 1]
```diff
@@ -597,3 +597,14 @@ def normalize(x):
             lazy = tree[branchname].lazyarray(keycache=keycache, basketcache=basketcache)
             for i in range(len(lazy), 0, -1):
                 self.assertEqual(normalize(lazy[i - 1 : i + 3]), strict[i - 1 : i + 3].tolist())
+
+    def test_hist_in_tree(self):
+        tree = uproot.open("tests/samples/Event.root")["T"]
+        check = [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0,
```
Comment (reikdas) [inline tests/test_tree.py]: Looks a little messy. Not sure how to improve this portion.

Comment (reikdas) [reply tests/test_tree.py]: Oh! I was just a little concerned that it was a bit too crude. You can merge it in if the code is okay :)

**Comment by jpivarski** [comment]
I'll accept it when the tests are in and positive.

**Pull request merged**